plugins/importexport/native/filter/NativeXmlSubmissionFilter.inc.php

```diff
@@ -70,6 +70,7 @@ function handleElement($node) {
 		$submissionDao = DAORegistry::getDAO('SubmissionDAO'); /* @var $submissionDao SubmissionDAO */
 		$submission = $submissionDao->newDataObject();
 
+		$submission->setData('locale', $node->getAttribute('locale') ?: $context->getPrimaryLocale());
```

Comment:
@jonasraoni is this visible to you?

Reply:
I did this modification because the `publications.locale` field isn't being used in `stable-3_3_0`, [its value is taken](https://github.com/pkp/pkp-lib/blob/stable-3_3_0/classes/publication/PKPPublicationDAO.inc.php#L55-L58) from the `submissions.locale` (which remains `null` without this modification and creates some issues).

The reasoning was kept in `main`, the `publications.locale` has been wiped off and now there's just the `submissions.locale`, so this behavior needs to be forwarded.

Reply:
Here's a related issue: https://github.com/pkp/pkp-lib/issues/7958